common/src/main/kotlin/org/bibletranslationtools/maui/common/io/IVersificationReader.kt

```diff
@@ -0,0 +1,7 @@
+package org.bibletranslationtools.maui.common.io
+
+import io.reactivex.Single
+
+interface IVersificationReader {
+    fun read(): Single<Map<String, List<Int>>>
```

Comment:
this apparently declares a public API. I would suggest having the return type as an object to encapsulate the components that make up a versification mapping at the corresponding level. Also, it makes the API method more readable - "Versification" (you name it) vs. `Map<String, List<Int>>`.